src/nrf5_sdk15_platform/communication/ruuvi_nrf5_sdk15_communication_ble_gatt.c

```diff
@@ -853,4 +859,43 @@ rd_status_t ri_gatt_dis_init (const ri_comm_dis_init_t * const p_dis)
     return ruuvi_nrf5_sdk15_to_ruuvi_error (err_code);
 }
 
+/**
+ * @brief Request connection parameter update for current connection.
+ *
+ * @param[in] params One of preset defaults: RI_GATT_TURBO, RI_GATT_STANDARD, RI_GATT_LOW_POWER.
+ * @retval RD_SUCCESS Parameter update was requested
+ * @retval RD_ERROR_INVALID_PARAM params was not one of supported defaults
+ * @retval RD_ERROR_INVALID_STATE if there is no ongoing GATT connection
+ * @retval Error code from BLE Stack if applicable
+ */
+rd_status_t ri_gatt_params_request (const ri_gatt_params_t params)
+{
+    ret_code_t err_code = NRF_SUCCESS;
+    ble_gap_conn_params_t gap_conn_params;
+    memset (&gap_conn_params, 0, sizeof (gap_conn_params));
+    gap_conn_params.slave_latency = SLAVE_LATENCY;
```

Comment:
Slave latency should be selected to match scenario. 0 on TURBO, "something reasonable" on STANDARD, maximum allowed by Apple on LOW_POWER. 

Determining the proper constants takes some research, they should be #define'd in header and reasoning for the value explained in comments

Reply:
Applied a calculator created by Silicon Labs: https://docs.silabs.com/resources/bluetooth/miscellaneous/mobile/selecting-suitable-connection-parameters-for-apple-devices/source/AppleConnectionParams_R11.xlsx